stable/traefik/values.yaml

```diff
@@ -1,6 +1,7 @@
 ## Default values for Traefik
 image: traefik
 imageTag: 1.7.20
+imagePullPolicy: IfNotPresent
```

Comment:
Can you comment this out, so that it is not required by default?

Reply:
I don't understand, you don't want to keep `IfNotPresent` by default ?

Reply:
we can keep, it makes sense